eZ/Publish/Core/Search/Legacy/Tests/Content/AbstractTestCase.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Search\Legacy\Tests\Content;
+
+use eZ\Publish\Core\Persistence\Legacy\Tests\Content\LanguageAwareTestCase;
+use eZ\Publish\Core\Persistence\Legacy\Content\Type\Gateway\DoctrineDatabase as ContentTypeGateway;
+use eZ\Publish\Core\Persistence\Legacy\Content\Type\Handler as ContentTypeHandler;
+use eZ\Publish\Core\Persistence\Legacy\Content\Type\Mapper as ContentTypeMapper;
+use eZ\Publish\Core\Persistence\Legacy\Content\FieldValue\Converter;
+use eZ\Publish\Core\Persistence\Legacy\Content\FieldValue\ConverterRegistry;
+use eZ\Publish\Core\Persistence\Legacy\Content\Type\Update\Handler as ContentTypeUpdateHandler;
+
+/**
+ * Abstract test suite for legacy search.
+ */
+class AbstractTestCase extends LanguageAwareTestCase
+{
+    private static $setup;
+
+    /**
+     * Field registry mock.
+     *
+     * @var \eZ\Publish\Core\Persistence\Legacy\Content\FieldValue\ConverterRegistry
+     */
+    private $converterRegistry;
+
+    /**
+     * @var \eZ\Publish\SPI\Persistence\Content\Type\Handler
+     */
+    private $contentTypeHandler;
+
+    /**
+     * Only set up once for these read only tests on a large fixture.
+     *
+     * Skipping the reset-up, since setting up for these tests takes quite some
+     * time, which is not required to spent, since we are only reading from the
+     * database anyways.
+     */
+    public function setUp()
+    {
+        if (!self::$setup) {
+            parent::setUp();
+            $this->insertDatabaseFixture(__DIR__ . '/../_fixtures/full_dump.php');
```

Comment:
Hmm, but then it means we're testing on SQLite only, since it is `eZ\Publish\Core\Search\Legacy\Tests\Content` namespace.

Update: ok... seems it was like that before in child class, so it's rather a follow up for later